src/python/pants/backend/awslambda/python/awslambda_python_rules.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from dataclasses import dataclass
+
+from pants.backend.awslambda.common.awslambda_common_rules import AWSLambdaTarget, CreatedAWSLambda
+from pants.backend.awslambda.python.lambdex import Lambdex
+from pants.backend.python.rules.pex import (
+  CreatePex,
+  Pex,
+  PexInterpreterConstraints,
+  PexRequirements,
+)
+from pants.backend.python.rules.pex_from_target_closure import CreatePexFromTargetClosure
+from pants.backend.python.subsystems.python_setup import PythonSetup
+from pants.backend.python.subsystems.subprocess_environment import SubprocessEncodingEnvironment
+from pants.engine.addressable import BuildFileAddresses
+from pants.engine.fs import Digest, DirectoriesToMerge
+from pants.engine.isolated_process import ExecuteProcessRequest, ExecuteProcessResult
+from pants.engine.legacy.structs import PythonAWSLambdaAdaptor
+from pants.engine.rules import UnionRule, optionable_rule, rule
+from pants.engine.selectors import Get
+
+
+@dataclass(frozen=True)
+class LambdexSetup:
+  requirements_pex: Pex
+
+
+@rule(name="Create Python AWS Lambda")
+async def create_python_awslambda(
+    lambda_tgt_adaptor: PythonAWSLambdaAdaptor,
+    lambdex_setup: LambdexSetup,
+    python_setup: PythonSetup,
+    subprocess_encoding_environment: SubprocessEncodingEnvironment) -> CreatedAWSLambda:
+  # TODO: We must enforce that everything is built for Linux, no matter the local platform.
```

Comment:
AFAIK, this should not be blocked? Can constrain to the relevant platform (cc @hrfuller ), and can pass the relevant args to pex.